source/YearlyOverview/YearlyOverview.html

```diff
@@ -10,7 +10,13 @@
     <body>
         <div id="header">
             <h1 id="index">
-                <a href="../Index/Index.html" id="back"> Index</a>
+                <a
+                    href="../Index/Index.html"
+                    style="text-decoration: none"
+                    id="back"
```

Comment:
You may need to pull from main as I think someone maybe Hunter, changed the code so there isn't CSS with the HTML which you branch still has